### DISCUSSION TITLE
Can't enable the plugin

### DISCUSSION THREAD

**Discussion by azanihs** [st3v3nmw/obsidian-spaced-repetition#134] [Q&A] [answered]
Hi,

 I have installed the plugin, but the plugin can't be enabled.  Every time I enable the plugin, the following message appears  " Failed to load plugin obsidian-spaced-repetition"

Does anyone have a similar issue?

Regards

**Comment by st3v3nmw** [comment, marked as answer]
Check if there's an error on the developer console and post it here.
This could have been easier to fix if you opened a bug report in the issues section.

**Reply by azanihs** [reply]
Here is the error message that I got:
![image](https://user-images.githubusercontent.com/687660/120956183-8fad2900-c785-11eb-9ce8-cd1212d2c794.png)
 
Ok, I will create a bug report for this.

**Reply by st3v3nmw** [reply]
That error had been reported before in the Obsidian forum. Your Obsidian installer version might be dated. Your can fix it by downloading it directly from the website and reinstalling.

**Reply by azanihs** [reply]
Solved.

I have reinstalled my obsidian, and now I can enable the plugin.

Thanks, Stephen
